src/test/java/our/yurivongella/instagramclone/service/PostServiceTest.java

```diff
@@ -120,13 +120,21 @@ public void readOnePost() {
     public void DeleteOnePost() {
         Long postId = postService.create(postCreateRequestDto);
 
-        System.out.println("Request UserId : "+SecurityUtil.getCurrentMemberId());
-
         postService.delete(postId);
 
         Assertions.assertThrows(
                 RuntimeException.class,
                 () -> postService.read(postId)
         );
     }
+
+    @DisplayName("특정 유저 게시물 리스트 블러오")
+    @Test
+    public void GetUsersPostList() {
+        Long postId = postService.create(postCreateRequestDto);
+
+        List<PostReadResponseDto> postlist = postService.getPostList(SecurityUtil.getCurrentMemberId());
+
+        assertThat(postlist.get(0).getAuthor().getId().equals(SecurityUtil.getCurrentMemberId()));
```

Comment:
동등 비교 테스트 할 때는 `assertThat(actual).isEqualTo(expected)` 형식으로 작성하는게 좋습니다.

```suggestion
        assertThat(postlist.get(0).getAuthor().getId()).isEqualTo(SecurityUtil.getCurrentMemberId());
```

Reply:
반영하여 수정하였습니다.
https://github.com/INSPJT/instagram_clone_backend/pull/35/commits/8f390e6b736e6ef2061583f6e87de5e300200f0c